ajax/libs/elfinder/package.json

```diff
@@ -24,7 +24,7 @@
       {
         "basePath": "",
         "files": [
-          "js/**/+(elFinder|elfinder|quicklook.googledocs)*.+(js|map)",
+          "js/**/*.+(html|js|map)",
```

Comment:
@dakshshah96 @cdnjs/intern3 Could you help me double check because there are too many files? 
Please~
I don't want to lost any file that CDNJS need to host.